modules/combined/test/tests/gap_heat_transfer_htonly/tests

```diff
@@ -1,67 +1,134 @@
 [Tests]
-
   [./3D]
     type = Exodiff
     input = 'gap_heat_transfer_htonly_test.i'
     exodiff = 'gap_heat_transfer_htonly_test_out.e'
     abs_zero = 1e-6
+    requirement = "Thermal contact shall solve plate heat transfer for a constant conductivity gap in 3D"
+    design = 'source/bcs/GapHeatTransfer.md'
+    issues = '#1609'
   [../]
-
   [./3D_Iters]
     type = Exodiff
     input = 'gap_heat_transfer_htonly_it_plot_test.i'
     exodiff = 'out_it_plot.e'
     custom_cmp = 'gap_heat_transfer_htonly_it_plot_test.cmp'
+    requirement = "Thermal contact shall solve plate heat transfer for a constant conductivity gap in 3D at each iteration"
```

Comment:
@permcody If I go back through the history you originally added this 3D_Iters test and I do not really know the difference between the 3D test above and this one other than in the Outputs block there is a parameter I have never seen or used before: `nonlinear_residual_dt_divisor`.  Perhaps you have a better idea of what the difference in requirements would be between this test and the one above it.

Reply:
Acknowledging that I've seen this. I probably won't have a chance to look at it until Monday.

Reply:
Any chance to take a look at the test and verify whether the requirement I have listed is sufficient?

Reply:
This is definitely not my test. It looks like I moved it way back in the module days but I can't seem to follow history before that. I might be @jasondhales test... 

At any rate, I think what you've done here is good and not worth holding up this PR any longer.